bodybuilder.d.ts

```diff
@@ -8,6 +8,12 @@ declare namespace bodybuilder {
 		agg: QuerySubFilterBuilder
 	) => QuerySubFilterBuilder;
 
+	export interface AggOptions {
+		name?: string;
```

Comment:
Is it possible to document these fields such that Intellisense displays a description of the usage for `_name`, `_meta`, etc.?

Reply:
Can do!

Reply:
Just pushed! I copied the description from the current docs, let me know if you'd like me to change anything